### PR TITLE
Install the Product Rating and Reviews Glue API

### PR DESCRIPTION
**Pull request by ievgen-malykhin**
Ticket: https://spryker.atlassian.net/browse/CC-16486

## PR Description

Update product feature and review feature integration guide

## Checklist
- [X] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 1]
```diff
@@ -0,0 +1,666 @@
...
+</details>
+
+{% endinfo_block %}
+
+#### Enable query expanders
+Activate the following plugins:
+
+| PLUGIN | SPECIFICATION | PREREQUISITES | NAMESPACE |
+| --- | --- | --- | --- |
+| FilterByIdProductReviewQueryExpanderPlugin |  | None | Spryker\Client\ProductReviewSearch\Plugin\Search |
```
Comment (dmiseev) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:218]: it's not related to GLUE API feature/IG, it's a part of `Product rating & reviews feature` integration .

Comment (ievgen-malykhin) [reply docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md]: The plugin was removed from this IG, _product-rating-and-reviews-feature-integration_ IG is pretty outdated and should be recreated considering a new template. FilterByIdProductReviewQueryExpanderPlugin should be mentioned there.

**Review by dmiseev — DISMISSED** [review]

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 2]
```diff
@@ -0,0 +1,636 @@
...
+To start feature integration, overview and install the necessary features:
+
+| NAME | VERSION | INTEGRATION GUIDE |
+| --- | --- | --- |
+| Spryker Core | {{page.version}} | [Feature API](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/glue-api-spryker-core-feature-integration.html) |
+| Product Rating & Reviews  | {{page.version}} |  |
+
+### 1) Install the required modules using Composer
+
+Run the following command to install the required modules:
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:28]:
```suggestion
```

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 3]
```diff
@@ -0,0 +1,636 @@
...
+
+| MODULE | EXPECTED DIRECTORY |
+| --- | --- |
+| ProductReviewsRestApi | vendor/spryker/product-reviews-rest-api |
+
+{% endinfo_block %}
+
+### 2) Set up database schema and transfer objects
+
+Run the following command to generate transfer changes:
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:45]:
```suggestion
Generate transfer changes:
```

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 4]
```diff
@@ -0,0 +1,636 @@
...
+
+| TRANSFER                               | TYPE | EVENT | PATH                                                                 |
+|----------------------------------------| --- | --- |----------------------------------------------------------------------|
+| RestProductReviewsAttributesTransfer   | class | created | src/Generated/Shared/Transfer/RestProductReviewsAttributesTransfer   |
+| BulkProductReviewSearchRequestTransfer | class | created | src/Generated/Shared/Transfer/BulkProductReviewSearchRequestTransfer |
+| StoreTransfer                          | class | created | src/Generated/Shared/Transfer/StoreTransfer                          |
+
+{% endinfo_block %}
+
+{% info_block warningBox "Verification" %}
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:64]:
```suggestion
```

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 5]
```diff
@@ -0,0 +1,636 @@
...
+Make sure that the following changes have been applied in the transfer objects:
+
+| TRANSFER                               | TYPE | EVENT | PATH                                                                 |
+|----------------------------------------| --- | --- |----------------------------------------------------------------------|
+| RestProductReviewsAttributesTransfer   | class | created | src/Generated/Shared/Transfer/RestProductReviewsAttributesTransfer   |
+| BulkProductReviewSearchRequestTransfer | class | created | src/Generated/Shared/Transfer/BulkProductReviewSearchRequestTransfer |
+| StoreTransfer                          | class | created | src/Generated/Shared/Transfer/StoreTransfer                          |
+
+{% endinfo_block %}
+
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:63]:
```suggestion
```

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 6]
```diff
@@ -0,0 +1,636 @@
...
+Make sure that the following changes have been applied in the transfer objects:
+
+| TRANSFER                               | TYPE | EVENT | PATH                                                                 |
+|----------------------------------------| --- | --- |----------------------------------------------------------------------|
+| RestProductReviewsAttributesTransfer   | class | created | src/Generated/Shared/Transfer/RestProductReviewsAttributesTransfer   |
+| BulkProductReviewSearchRequestTransfer | class | created | src/Generated/Shared/Transfer/BulkProductReviewSearchRequestTransfer |
+| StoreTransfer                          | class | created | src/Generated/Shared/Transfer/StoreTransfer                          |
+
+{% endinfo_block %}
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:62]:
```suggestion
```

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 7]
```diff
@@ -0,0 +1,636 @@
...
+| SpyProductAbstractStorage | class | extended |src/Orm/Zed/ProductStorage/Persistence/Base/SpyProductAbstractStorage | syncPublishedMessageForMappings(), syncUnpublishedMessageForMappings() |
+| SpyProductConcreteStorage | class | extended | src/Orm/Zed/ProductStorage/Persistence/Base/SpyProductConcreteStorage | syncPublishedMessageForMappings(), yncUnpublishedMessageForMappings() |
+
+{% endinfo_block %}
+
+### 3) Set up behavior
+
+#### Reload data to storage
+
+Run the following commands to reload abstract and product data to storage.
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:79]:
```suggestion
Reload abstract and product data to storage.
```

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 8]
```diff
@@ -0,0 +1,636 @@
...
+    "links": {
+        "self": "http://glue.de.suite-nonsplit.local/abstract-products/139/product-reviews?page[offset]=0&page[limit]=5",
+        "last": "http://glue.de.suite-nonsplit.local/abstract-products/139/product-reviews?page[offset]=0&page[limit]=5",
+        "first": "http://glue.de.suite-nonsplit.local/abstract-products/139/product-reviews?page[offset]=0&page[limit]=5"
+    }
+}
+```
+</details>
+
+{% endinfo_block %}
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:212]:
```suggestion
```

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 9]
```diff
@@ -0,0 +1,636 @@
...
+
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:213]:
```suggestion
```

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 10]
```diff
@@ -0,0 +1,636 @@
...
+{% info_block warningBox "Verification" %}
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:214]:
```suggestion
```

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 11]
```diff
@@ -0,0 +1,636 @@
...
+            "links": {
+                "self": "http://glue.de.suite-nonsplit.local/product-reviews/26"
+            }
+        }
+    ]
+}
+```
+</details>
+
+{% endinfo_block %}
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:483]:
```suggestion
```

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 12]
```diff
@@ -0,0 +1,636 @@
...
+
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:484]:
```suggestion
```

File: docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md [hunk 13]
```diff
@@ -0,0 +1,636 @@
...
+{% info_block warningBox "Verification" %}
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md:485]:
```suggestion
```

**Review by VadymSachenko — CHANGES_REQUESTED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**